docs/getting-started.md

```diff
@@ -0,0 +1,106 @@
+## Getting Started
+
+#### Configuring Migrate for Compute Engine Landing Zone with CFT
+
+### Tools
+
+**Install Cloud SDK** \
+The Google Cloud SDK is used to interact with your GCP resources. [Installation instructions](https://cloud.google.com/sdk/downloads) for
+
+multiple platforms are available online.
+
+**Install Terraform** \
+Terraform is used to automate the manipulation of cloud infrastructure. Its [installation instructions](https://www.terraform.io/intro/getting-started/install.html) are also available online.
+
+**Authenticate gcloud** \
+Prior to running this, ensure you have authenticated your gcloud client by running the following command:
+
+```
+gcloud auth application-default login
+```
+
+### Deployment
+
+Download the repository:
+
+```
+git clone https://github.com/terraform-google-modules/terraform-google-migrate.git
+```
+
+File structure
+
+The project has the following folders and files:
+
+```
+
+/: root folder
+/examples: Examples for doing single and multi project deployments
+/modules: Modules for core, single and multi projects
+/helpers: Optional helper scripts for ease of use
+/main.tf: TODO
+/variables.tf:TODO
+/output.tf: TODO
+/readme.md: this file
+
+```
+
+**Single project deployment:**
```

Comment:
Guide should focus on multi-project use case.

Reply:
Should I remove the single project deployment docs from this or should I move it below the multi deployment use case?

Reply:
Move it into a separate doc.